lib/config.ts

```diff
@@ -97,6 +99,10 @@ export class AppConfig {
 
       console.info('\nConfig loaded from bewcloud.config.ts', JSON.stringify(this.config, null, 2), '\n');
 
+      if (this.config.email.port !== 465 && this.config.email.tlsMode === null) {
+        console.warn('Config WARN: When using `.email.port` with a value other than 465, please set `.email.tlsMode` to either "starttls" or "none" to explicitly enable or disable usage of StartTLS! Support for legacy opportunistic StartTLS will be removed in bewCloud 4!');
```

Comment:
Please make this more consistent with past warnings.

```suggestion
        console.warn('DEPRECATION WARNING: When using `config.email.port` with a value other than `465`, please set `config.email.tlsMode` to either `'starttls'` or `'none'` to explicitly enable or disable usage of StartTLS! Support for legacy opportunistic StartTLS will be removed in bewCloud 4!');
```